server/blob/blob-cassandra/src/test/java/org/apache/james/blob/cassandra/cache/DumbBlobStoreCacheContract.java

```diff
@@ -35,7 +35,7 @@
 
 public interface DumbBlobStoreCacheContract {
 
-    byte[] EIGHT_KILOBYTES = Strings.repeat("01234567\n", 1024).getBytes(StandardCharsets.UTF_8);
+    byte[] EIGHT_KILOBYTES = Strings.repeat("0123456\n", 1000).getBytes(StandardCharsets.UTF_8);
```

Comment:
Revert that change and change your default configuration to 8 * 1024 instead

Reply:
this will be `9000` when 8 * 1024 ~ 8k9

Reply:
1 KILOBYTES is 1024 bytes

Google answerd this:

```
The kilobyte has traditionally been used to refer to 1024 bytes (210 B), a usage still common. The usage of the metric prefix kilo for binary multiples arose as a convenience, because 1024 is approximately 1000. ... The binary meaning of the kilobyte for 1024 bytes typically uses the symbol KB, with an uppercase letter K.
```

We need to be strict and stick to 1024

Reply:
And google is wrong : https://en.wikipedia.org/wiki/Kibibyte